### DISCUSSION TITLE
Candle Discord?

### DISCUSSION THREAD

**Discussion by eisenzopf** [huggingface/candle#2643] [General]
What happened to the #candle discord channel on HF Discord server?

**Comment by freedomtowin** [comment]
I'd also like to join.

**Comment by jermsam** [comment]
yes, how do i join?

**Comment by aaelony** [comment]
+1 from to join as well... there may be many more (?)

**Comment by danielclough** [comment]
There used to be a channel on Hugging Face Discord, but I lost it.
I just created a Server so that we can chat and learn from one another.

If you have contributed to Candle and want to be an admin let me know.

Edit: 
Since the official server has been found (link below) we are going to focus on "Full Stack Rust AI/ML".

https://discord.gg/4Uv5sHM3AA

**Comment by LaurentMazare** [comment]
It seems that the candle channel is still available on the HF server (at least I can see it).
<img width="678" alt="image" src="https://github.com/user-attachments/assets/e42e9589-f528-4795-83d0-dc705f8b5599" />

**Reply by danielclough** [reply]
I found it again!
I looked real hard when I first saw this Github post and couldn't find it.
So much going on in there. :dizzy_face: 

It's under `OPEN SOURCE ML`

Correct link below.

**Reply by jermsam** [reply]
<img width="1641" alt="Screenshot 2025-03-16 at 7 36 31 AM" src="https://github.com/user-attachments/assets/ab7c5ec2-b5ca-4113-bea9-30079c975743" />
it's giving me that error

**Reply by aaelony** [reply]
For me, that link leads to the following error message: 


NO TEXT CHANNELS
You find yourself in a strange place. You don't have access to any text channels, or there are none in this server.

**Comment by AlpineVibrations** [comment]
https://discord.com/channels/879548962464493619/1136218819447238726

**Reply by aaelony** [reply]
![image](https://github.com/user-attachments/assets/689a3098-f167-4ad4-9423-5ac1fb145792)

**Comment by AlpineVibrations** [comment]
https://discord.gg/hugging-face-879548962464493619

**Reply by aaelony** [reply]
that worked, thank-you!
